### ISSUE TITLE
Add Logging or Indicator when Facts Cache is Cleared

### ISSUE THREAD

**Issue by jamesmarshall24** [ansible/awx#9798]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY

Add an indicator in Tower activity stream or in logging when fact caching for a host is cleared.

The meta module does not appear to be logged at all in the job_events endpoint, so there is no way to tell when they were cleared unless you confirm facts list is empty at the same time as checking `ansible_facts_modified` for the host. Otherwise, there is no record of the event.

**Comment by wenottingham** [comment]
This is already logged:

https://github.com/ansible/awx/blob/devel/awx/main/models/jobs.py#L894

Activity stream is the wrong place for this (you don't want 10000 entries for a large inventory

**Comment by jamesmarshall24** [comment]
> This is already logged:
> 
> https://github.com/ansible/awx/blob/devel/awx/main/models/jobs.py#L894
> 
> Activity stream is the wrong place for this (you don't want 10000 entries for a large inventory

@wenottingham 
Looks like this just goes to an external logger through the log aggregator? I'm not seeing an indicator of facts being cleared in the k8s/ocp logs at all.

**Comment by wenottingham** [comment]
Correct, it goes to the external logger under 'system tracking' category.

**Comment by wenottingham** [comment]
We could consider adding this to the service logs, but you can get this via external logging now.
